docs/UserGuide.md

```diff
@@ -3,14 +3,37 @@ layout: page
 title: User Guide
 ---
 
-Where Got Time (WGT) **a perfect desktop app** dedicate to managing your events and plan out your meetings with your friends and family. It is developed for university student, who can type fast to efficiently keep track of all of their events and their friends' events **via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, WGT can help you find a date that all your friends are free to meet instead of having to manually compare timetables with each other.
-
-* Table of Contents
-{:toc}
+Where Got Time (WGT) **a perfect desktop app** dedicate to managing your events and plan out your meetings with your 
+friends and family. It is developed for university student, who can type fast to efficiently keep track of all of their 
+events and their friends' events **via a Command Line Interface (CLI)** while still having the benefits of a 
+Graphical User Interface (GUI). If you can type fast, WGT can help you find a date that all your friends are free to 
+meet instead of having to manually compare timetables with each other.
+
+* [Quick Start](#QuickStart) 
+* [Features](#Features)
+  * [Person-related commands](#PersonRelatedCommands)
+    * [Add person's information into WGT](#AddPerson)
+    * [List all the persons stored in WGT](#ListPerson)
+    * [Edit information about a person](#EditPerson)
+    * [Find a person's information by his/her name](#FindPerson)
+    * [Delete a person's information in WGT](#DeletePerson)
+  * [Group-related commands](#GroupRelatedCommands)
+    * [Create a group in WGT](#CreateGroup)
+    * [Delete a group created in WGT](#DeleteGroup)
+    * [List all the groups created in WGT](#ListGroup)
+    * [Find a group by the group's name](#FindGroup)
+  * [Event-related commands](#EventRelatedCommands)
+    * [Create an event](#CreateEvent)
+    * [Delete an event](#DeleteEvent)
+    * [Edit information about an event](#EditEvent)
+    * [List all the events created](#lListEvent)
+  * [Miscellaneous features](#MiscellaneousCommands)
+    * [Clear all data stored in WGT](#Clear)
+    * [Exit the program](#Exit)
+    * [Help](#Help)
```

Comment:
Im not sure if we need this because the table of contents in 8 and 9 already have navigation to the respective sections. It cant be seen on GitHub but can be seen on our website page https://ay2223s2-cs2103t-t09-2.github.io/tp/UserGuide.html

Also i tried clicking the hyperlinks and it doesn't navigate to the respective sections